examples/cat_facts/.config/nextest.toml

```diff
@@ -0,0 +1,6 @@
+[profile.ci]
```

Comment:
Is this needed now there is one at the repo root?

Reply:
Unfortunately yes - it looks for this relative to the workspace `Cargo.toml`.  Did look to see if there was an env var we could set but couldn't see one.

We _could_ pass in a `--config-path` to each command though if we'd rather...